src/libANGLE/renderer/d3d/d3d11/SwapChain11.cpp

```diff
@@ -69,6 +69,7 @@ SwapChain11::SwapChain11(Renderer11 *renderer,
       mPassThroughResourcesInit(false),
       mNativeWindow(nativeWindow),
       mFirstSwap(true),
+      mSwapEffectDiscards(false),
```

Comment:
Utter nitpick but I'd have gone with a name that signifies the bool-ness of this variable more to me, such as mIsSwapEffectDiscard. However, if the -s subject-verb agreement is a usual naming convention for bools in the codebase, never mind my comment.

Reply:
Make sense: 2ae3aa286addf6468b725ace8e34a4d2f2ede310